CMake/Dependencies/libkvspic-CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ include(ExternalProject)
 # clone repo only
 ExternalProject_Add(libkvspic-download
 	GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-pic.git
-	GIT_TAG           b9EfgQfFJfESh4UPThMStqqKxfe0e+AWqkQn1EOX
+	GIT_TAG           c9e4b524ca80758c2a3082ffe98cb1e18dcebea8
```

Comment:
That commit is from this branch:

https://github.com/awslabs/amazon-kinesis-video-streams-pic/compare/no_tags_before_cluster

Should we first merge the pic change into develop and then update this hash to be the one from develop?  Not sure we should be referencing a commit from this branch and merging into develop.

In the past we had an issue where a commit disappeared I think what happened there was we referenced a commit from a branch but after merge that branch gets deleted along with this commit hash.

Reply:
I agree. Do not merge this branch until the corresponding PIC branch has been merged and the git hash here has been updated.